src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/IssueShell6738.cs

```diff
@@ -0,0 +1,43 @@
+#if TEST_FAILS_ON_WINDOWS && TEST_FAILS_ON_ANDROID
```

Comment:
There is a parameter to include or not the TitleBar: https://github.com/dotnet/maui/blob/8108c344e2d6e5399b89f9acd9b852f1d9f25eb6/src/Controls/tests/TestCases.Shared.Tests/UITest.cs#L186
Could you try it?